src/Core/maps/src/Handlers/Map/MapHandler.Android.cs

```diff
@@ -406,6 +412,13 @@ void OnInfoWindowClick(object? sender, GoogleMap.InfoWindowClickEventArgs e)
 		void OnMapClick(object? sender, GoogleMap.MapClickEventArgs e) =>
 			VirtualView.Clicked(new Devices.Sensors.Location(e.Point.Latitude, e.Point.Longitude));
 
+		string? GetMapId()
+		{
+			int resourceId = Context.Resources?.GetIdentifier("map_id", "string", Context.PackageName) ?? 0;
```

Comment:
Is `map_id` an official Google recommendarion?

Reply:
Yes https://developers.google.com/maps/documentation/get-map-id#add-a-map-id-to-your-app

Reply:
That looks more like an example, what happens if you have 2 map controls with different ids? Or is that never possible?